PayPalNativeCheckout/src/test/java/com/braintreepayments/api/PayPalNativeCheckoutInternalClientUnitTest.java

```diff
@@ -450,7 +450,7 @@ public void sendRequest_withPayPalCheckoutRequest_callsBackPayPalResponseOnSucce
         assertFalse(payPalResponse.isBillingAgreement());
         assertEquals("authorize", payPalResponse.getIntent());
         assertEquals("sample-merchant-account-id", payPalResponse.getMerchantAccountId());
-        assertEquals("EC-HERMES-SANDBOX-EC-TOKEN", payPalResponse.getPairingId());
+        assertNull(payPalResponse.getPairingId());
```

Comment:
This helped clear the test, and also makes me wonder if we should be collecting data for MXO flows? The pairing ID relies on the presence of an app switch URL. It seems like we're collecting data for Native PayPal only because the implementation of `PayPalNativeCheckoutInternalClient` is identical to web.